bulk_deploy/tsuru_api_service.rb

```diff
@@ -0,0 +1,119 @@
+require_relative './../dsl/ssh_helper'
+
+class TsuruAPIService
+
+  def initialize(logger:, api_client:, tsuru_home:)
```

Comment:
What command are you running? Did you cd into `bulk_deploy` directory?

I am recreating my environment so can test once my new env is up and running.